DEVDOCS-2528: Embedded Checkout Domain Setup (#970)
bigcommerce/dev-docs: docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

<!-- theme:  -->

> > Channels created via API are visible in the BigCommerce store's Control Panel in **Products** > **Listed On**. The Orders section will now also include a filter for your channel.

Reviewer: > Channels created via API are visible in the BigCommerce store's control panel in **Products** > **Listed On**. The Orders section will now also include a filter for your channel.

## Creating a cart

> To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api)  `/cart` endpoint.

Reviewer 2: ```suggestion
To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/store-management/carts)  `/cart` endpoint.
```

> To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api)  `/cart` endpoint.

Reviewer: To proceed to checkout, we'll need an active cart. To create one, send a `POST` request to the [Server-to-Server Cart API's](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api) `/cart` endpoint.

```

### Redirecting a logged-in customer to embedded checkout

For some use cases, you may want your customer to log in before they can begin the checkout process.

> Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer).

Reviewer 2: ```suggestion
Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/storefront/customer-login-api).
```

> Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer).

Reviewer: Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/storefront/customer-login-api).

### Are hosted payment gateways supported with Embedded Checkout?
At this time, you cannot embed a checkout using a hosted payment gateway. See [Available Payment Gateways](https://support.bigcommerce.com/s/article/Available-Payment-Gateways#all-available) to determine which type of gateway you're using.

> ### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**

Reviewer 2: ```suggestion
### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Error?**
```

> ### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**

Reviewer: ### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" errors?

> ### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?**

Reviewer 3: ```suggestion
### How do I resolve Embedded Checkout 403 "Cannot start checkout session with an empty cart" Errors?
```

> For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center.

Reviewer 3: ```suggestion
For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari).
```

> In order for Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatibile with Safari.

Reviewer 2: ```suggestion
For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. For more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari, see [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center. 
```

> In order for Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), your checkout page must be served from the same domain as your BigCommerce storefront. For example, if your headless storefront is `www.mystore.com`, then your BigCommerce store's domain should be `checkout.mystore.com`. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatibile with Safari.

Reviewer: For Embedded Checkout to work correctly for shoppers using a browser with restricted privacy settings (like Apple's Safari), you must serve your checkout page from the same domain as your BigCommerce storefront. For example, if your headless storefront is www.mystore.com, your BigCommerce store's domain should be checkout.mystore.com. See [BigCommerce for WordPress](https://support.bigcommerce.com/s/article/BigCommerce-for-WordPress-Checkout#safari) in the Help Center for more information on making Embedded Checkout on a headless WordPress storefront compatible with Safari.
